src/Helmet.js

```diff
@@ -163,20 +163,31 @@ const updateTitle = title => {
 
 const updateHtmlAttributes = (attributes) => {
     const htmlTag = document.getElementsByTagName("html")[0];
-
-    const oldAttributeCount = htmlTag.attributes.length;
-    if (oldAttributeCount) {
-        for (let i = oldAttributeCount - 1; i >= 0; i--) {
-            htmlTag.removeAttribute(htmlTag.attributes[i].name);
-        }
-    }
-
+    const helmetAttr = htmlTag.getAttribute(HELMET_ATTRIBUTE);
```

Comment:
How about something like `helmetAttributeString`? It's really just `helmetAttributes` before it gets split into an array